src/Orchard.Web/Modules/Orchard.Layouts/Controllers/BlueprintAdminController.cs

```diff
@@ -43,12 +44,19 @@ public class BlueprintAdminController : Controller, IUpdateModel {
             _shapeFactory = shapeFactory;
             _transactionManager = transactionManager;
             _signals = signals;
+            Services = orchardServices;
+
             T = NullLocalizer.Instance;
         }
 
+        public IOrchardServices Services { get; set; }
```

Comment:
Out of curiosity, why is this a public property instead of a private field? I've seen this pattern elsewhere, but never understood why.